sky/tools/create_macos_framework.py

```diff
@@ -114,6 +119,29 @@ def process_framework(dst, args, fat_framework, fat_framework_binary):
 
     subprocess.check_call(["strip", "-x", "-S", fat_framework_binary])
 
+  macos_filepath_with_entitlements = ''
+  macos_filepath_without_entitlements = 'FlutterMacOS.framework.zip/Versions/A/FlutterMacOS'
+
+  embed_codesign_configuration(
+      os.path.join(dst, 'entitlements.txt'), macos_filepath_with_entitlements
+  )
+
+  embed_codesign_configuration(
+      os.path.join(dst, 'without_entitlements.txt'),
+      macos_filepath_without_entitlements
+  )
+
+  # Zip FlutterMacOS.framework.
+  subprocess.check_call([
```

Comment:
Is this the resolution for https://github.com/flutter/engine/pull/35623 that was reverted here https://github.com/flutter/engine/pull/35680, because it was expected to not have entitlements?  I'm not sure how these `without_entitlements` files work.

https://github.com/flutter/flutter/blob/fefb2b00bab8548e66aa09849dbd51c4b9c357d1/dev/conductor/core/lib/src/codesign.dart#L188

@christopherfujino

Reply:
Sorry didn't realize there is an engine PR review on Thursday. These `without_entiltements` and 'entitlements.txt` are just text files embedded in the zip artifacts, not going to affect the correctness of anything.

Reply:
Inspecting locally, looks like the symlinks are dropped in the build process. Current assumption is based on https://github.com/flutter/engine/pull/35673.

Reply:
Is this ready for review?